src/coreclr/src/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -3915,6 +3915,7 @@ private sealed class ActivatorCache
             internal readonly RuntimeMethodHandleInternal _hCtorMethodHandle;
             internal MethodAttributes _ctorAttributes;
             internal CtorDelegate? _ctor;
+            internal ObjectFactory? _objectFactory;
```

Comment:
This does not really need to go through the extra objectFactory indirection. The allocator and constructor pointers can be stored directly here.

Reply:
`ObjectFactory` is gone as of the latest iteration, and we use the fnptrs directly.